### PR TITLE
MULE-16463: Remove `flatMap` in static <flow-ref>	

### PR DESCRIPTION
**Pull request by plbalbi**

### CODE REVIEW

**Comment by plbalbi** [comment]
--validate

**Comment by plbalbi** [comment]
--valite

**Comment by plbalbi** [comment]
--validate

**Comment by plbalbi** [comment]
--validate

**Comment by plbalbi** [comment]
--validate

File: core/src/main/java/org/mule/runtime/core/internal/construct/AbstractPipeline.java [hunk 1]
```diff
@@ -226,7 +226,7 @@ private ReactiveProcessor dispatchToFlow() {
 
   protected Function<CoreEvent, Publisher<? extends CoreEvent>> routeThroughProcessingStrategy() {
     return event -> {
-      // Retrieve response publisher before error is communicated
+      // Retrieve response publisher before error is communicate
```
Comment (elrodro83) [inline core/src/main/java/org/mule/runtime/core/internal/construct/AbstractPipeline.java]: this was ok before

**Pull request merged**